src/main/java/com/cinderella/dao/mysql/MySQLConnection.java

```diff
@@ -230,7 +275,21 @@ public boolean updateManger(Manager manager) {
 	 */
```

Comment:
Please add a comment/TODO to explain that this functionality may not be used in the current phase but will be considered later.